<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarrerasWindow</class>
 <widget class="QMainWindow" name="CarrerasWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>710</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelProgramasPregrado">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Programas de Pregrado</string>
    </property>
   </widget>
   <widget class="QWidget" name="W_Inicio" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonVolver">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Inicio</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/images/imagenes/Imagen1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="QWFacultadArteyDiseno" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelFacultad3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Facultad de Arte Y Dise√±o</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetCarreras_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>381</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="QWFacultadIngenieria" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listWidgetCarreras">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>371</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFacultad2_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Facultad de Ingenieria </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="QWFacultad" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelFacultad1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Facultad de Ciencias Administrativas y Sociales</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetCarrera">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>361</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>QWFacultadArteyDiseno</zorder>
   <zorder>labelProgramasPregrado</zorder>
   <zorder>W_Inicio</zorder>
   <zorder>QWFacultadIngenieria</zorder>
   <zorder>QWFacultad</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
